charts/admission-controller/templates/webhook/admissionregistration.yaml

```diff
@@ -4,115 +4,27 @@ so the template is executed just once
  */}}
 {{- $certString := include "admissionController.webhook.gen-certs" . -}}
 {{- $certList := split "$" $certString -}}
+
 ---
 {{- $existingVac := (lookup "admissionregistration.k8s.io/v1" "ValidatingWebhookConfiguration" (include "admissionController.namespace" .) (include "admissionController.webhook.fullname" .))}}
 {{- $existingVacMetadataAnnotations := (default dict (default dict $existingVac.metadata).annotations) }}
 {{- $existingVacMetadataAnnotationsReleaseName := (default "" (index $existingVacMetadataAnnotations "meta.helm.sh/release-name")) }}
 {{- $existingVacMetadataAnnotationsReleaseNamespace := (default "" (index $existingVacMetadataAnnotations "meta.helm.sh/release-namespace")) }}
-{{- if (or (not $existingVac) (and $existingVac (eq $existingVacMetadataAnnotationsReleaseName .Release.Name ) (eq $existingVacMetadataAnnotationsReleaseNamespace .Release.Namespace ))) }}
-apiVersion: admissionregistration.k8s.io/v1
-kind: ValidatingWebhookConfiguration
-metadata:
-  name: {{ include "admissionController.webhook.fullname" . }}
-  namespace: {{ include "admissionController.namespace" . }}
-webhooks: []
-{{- end}}
----
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   name: {{ include "admissionController.webhook.fullname" . }}
   namespace: {{ include "admissionController.namespace" . }}
+{{- if (or (not $existingVac) (and $existingVac (eq $existingVacMetadataAnnotationsReleaseName .Release.Name ) (eq $existingVacMetadataAnnotationsReleaseNamespace .Release.Namespace ))) }}
+{{- include "admissionController.webhookTemplate"  (merge (dict "Cert" $certList._2) . ) }}
+{{- else }}
```

Comment:
We prevent duplicate resource as described in #1310 by either using post-install hook (if we detect the VAC already exists but doesn't have the proper helm lifecycle annotations), or using a standard resource without hook if the VAC doesn't exist, or it exists and has the proper annotations (clean installs, or resource already annotated)